src/VisualStudio/Core/Def/project.json

```diff
@@ -78,6 +82,22 @@
       "version": "14.3.25407",
       "suppressParent": "all"
     },
+    "Microsoft.VisualStudio.TextManager.Interop.10.0": {
+      "version": "10.0.30319",
+      "suppressParent": "all"
```

Comment:
Why are these `"suppressParent": "all"`? I understand the RoslynDependencies* ones might be due to us not having proper packages so they can't "flow" like a NuGet package should and handle versioning with higher versioned and shipped packages.

(I'm not asking because I think it's _wrong_, it's just because I can't read your mind.)